tests/implementor/conftest.py

```diff
@@ -325,6 +324,55 @@ def train_speed_fault_configuration(train_speed_fault_configuration_data):
     return TrainSpeedFaultConfiguration.create(**train_speed_fault_configuration_data)
 
 
+# ------------- TrackBlockedFaultConfiguration ----------------
+
+
+@pytest.fixture
+def track_blocked_fault_configuration_data(track: Track) -> dict:
```

Comment:
Because you probably remove this line in the other pr before creating this branch, you have to delete it here too.